api_specifications/filestore-1.1.yaml

```
@@ -0,0 +1,387 @@
+                $ref: '#/components/schemas/UploadFileResponse'
+        400:
+          description: Error response in case of failures.
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/ErrorRes'
+  
+  /delete:
+    post:
+      tags:
+      - Delete
```

yamllint

[error] wrong indentation: expected 8 but found 6

(indentation)